controllers/apply/forms/reaching-communities-form.js

```diff
@@ -297,12 +296,6 @@ formModel.registerSuccessStep({
                                 .then(() => resolve(formData));
                         })
                         .catch(err => {
-                            Raven.captureMessage('Error converting template to inline CSS', {
-                                extra: err,
-                                tags: {
-                                    feature: 'reaching-communities'
-                                }
-                            });
                             return reject(err);
```

Comment:
Does this mean we won't know anymore if the email rendering failed?

Reply:
It gets captured in the form-router catch step instead.